json5/model.py

```diff
@@ -46,6 +46,28 @@ def __init__(self, *key_value_pairs, trailing_comma=None, leading_wsc=None, tok=
         assert leading_wsc is None or all(isinstance(item, str) or isinstance(item, Comment) for item in leading_wsc)
         super().__init__(key_value_pairs=key_value_pairs, trailing_comma=trailing_comma, leading_wsc=leading_wsc or [], tok=tok)
 
+    def find_key_value_pair(self, key):
```

Comment:
Are we expecting that keys are always Python strings?

I think we expect values to all be the `Value` model type. Would it be reasonable to expect given keys to also be of type `Key`? 

See also remarks on `__setitem__` -- perhaps same idea could apply here? Maybe we accept both?

Reply:
Similar to other discussion, I think this could be resolved at a later time.